<launch>

  <arg name="marker_size" default="13.35" />
  <arg name="max_new_marker_error" default="0.08" />
  <arg name="max_track_error" default="0.2" />
  <arg name="cam_image_topic" default="/cameras/right_hand_camera/image" />
  <arg name="cam_info_topic" default="/cameras/right_hand_camera/camera_info" />
  <arg name="output_frame" default="/right_hand_camera" />
  <arg name="mname" default="baxter_right_hand_camera" />


  <node name="tester" pkg="final" type="right_hand_camera.py" respawn="false" output="screen" /> 

</launch>

